cdap-docs/admin-manual/source/_includes/installation/configuration.txt

```diff
@@ -48,6 +48,14 @@ for CDAP to run successfully.
      $ sudo cp -f /etc/cdap/conf.mycdap/cdap-site.xml.example /etc/cdap/conf.mycdap/cdap-site.xml
      $ sudo vi /etc/cdap/conf.mycdap/cdap-site.xml
 
+#. If necessary, configure a ``cdap-env.sh`` after you have installed the CDAP packages.
+
+   Environment variables that will be included in the environment used when launching CDAP and
+   programs can be set in the ``cdap-env.sh`` file, usually at ``/etc/cdap/conf.mycdap/cdap-env.sh``.
+   
+   This only needed if you need to customize the environment launching CDAP and components, such
```

Comment:
"This **is** only..."  (and maybe "This is only **necessary** if you need to... ?)